rusoto/core/src/region.rs

```diff
@@ -43,7 +43,7 @@ pub enum Region {
     /// Region that covers West Europe
     EuWest2,
 
-    /// Region that covers South-East Europe
+    /// Region that covers South-East part of America
```

Comment:
Can we change this to `Region that covers South-East Americas`?  This can make it clearer it's not just America as in the United States of America. 😄

Reply:
Yeah, that would make things much clearer, in the initial request I kept (uneasily) to the original langauge.